src/test/scala/de/upb/cs/uc4/hyperledger/testBase/TestBaseProductionNetwork.scala

```diff
@@ -3,11 +3,11 @@ package de.upb.cs.uc4.hyperledger.testBase
 import java.nio.file.{ Path, Paths }
 
 protected class TestBaseProductionNetwork extends TestBaseTrait {
-  val networkDescriptionPath: Path = Paths.get(getClass.getResource("/connection_profile_kubernetes.yaml").toURI)
+  val networkDescriptionPath: Path = Paths.get("./hlf-network/assets/connection_profile_kubernetes_local.yaml")

```

Comment:
Maybe not perfect for development, maybe use a default path and allow overwriting?